tests/test_tpa_pipeline.py

```diff
@@ -6,29 +6,17 @@
 
 import unittest
 
-from django.http import HttpResponseRedirect
 import mock
 from pytest import mark, raises
 
-from enterprise.models import (
-    EnterpriseCustomer,
-    EnterpriseCustomerUser,
-    UserDataSharingConsentAudit,
-)
-
-from enterprise.tpa_pipeline import (
-    get_enterprise_customer_for_request,
-    get_ec_for_running_pipeline,
-    get_enterprise_customer_for_sso,
-    active_provider_enforces_data_sharing,
-    active_provider_requests_data_sharing,
-    get_consent_status_for_pipeline,
-    verify_data_sharing_consent,
-    set_data_sharing_consent_record,
-)
+from django.http import HttpResponseRedirect
 
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser, UserDataSharingConsentAudit
+from enterprise.tpa_pipeline import (active_provider_enforces_data_sharing, active_provider_requests_data_sharing,
+                                     get_consent_status_for_pipeline, get_ec_for_running_pipeline,
+                                     get_enterprise_customer_for_request, get_enterprise_customer_for_sso,
+                                     set_data_sharing_consent_record, verify_data_sharing_consent)
```

Comment:
I would agree; these imports are generated by isort. When it complains about import sorting in within a particular line, it does so with pretty much no additional information, so the easiest way to make it happy is to tell it to fix things itself.

Of course, now I actually look at it, it seems like it's just sorting the items alphabetically, so I can make that change manually from here on out, resulting in nice single-line imported items.